client/src/components/profile-page/Edit-Profile.css

```diff
@@ -0,0 +1,56 @@
+.edit-page {
+  position: relative;
+  background-color: var(--light-yellow);
+}
+
+.blur-effect {
+  filter: blur(3px);
+}
+
+.edit-pen {
+  position: absolute;
+  color: red;
+  z-index: 10;
+  font-size: 30px;
+  left: 116.84px;
+  top: 280.45px;
+}
+
+.edit-photo {
+  position: absolute;
+  color: red;
+  z-index: 10;
+  font-size: 30px;
+  left: 56px;
+  top: 110.2px;
+}
+
+.edit-title {
+  position: absolute;
+  left: 136.5px;
+  top: 40px;
+}
+.edit-page {
+  position: relative;
+}
+
+.edit-title {
+  position: absolute;
+  left: 136.5px;
+  top: 40px;
+}
```

Comment:
Hm. I see a lot of relative/absolute positioning here (with custom left and top).
This might cause some trouble whenever the user has a slightly different screen size. 

You should use grid and flex for layouting. And if you use absolute positioning, you should use relative units (e.g. percentages) more than fixed pixels.

In this case it might be fine because you are targetting "iphones" only so to say, but it's something you should consider, and maybe think if there is a better way without so much absolute positioning.

Keep it in mind